modules/enabling-custom-images.adoc

```diff
@@ -0,0 +1,31 @@
+:_module-type: PROCEDURE
+
+[id='enabling-custom-images_{context}']
+= Enabling custom images in {productname-short}
+
+All {productname-short} admin users can import custom workbench images, by default, by selecting the *Settings* -> *Notebook images* navigation option in the {productname-short} dashboard.
+
+If the *Settings* -> *Notebook images* option is not available, check the following settings, depending on which navigation element does not appear in the dashboard:
+
+* The *Settings* menu does not appear in the {productname-short} navigation bar
++
+The visibility of the OpenShift AI dashboard’s Settings menu is determined by your user permissions. By default, the Settings menu is available to OpenShift AI administration users (users that are members of the rhoai-admins group). Users with the OpenShift  cluster-admin role, are automatically added to the rhoai-admins group and are granted administrator access in OpenShift AI. 
```

Comment:
Replace "OpenShift AI" with "{productname-short}", and bold *Settings * menu. 

We have attributes for the admin group, {odh-admin-group} and {oai-admin-group} (which we really should change to be the same attribute name upstream and downstream).